receiver/kafkametricsreceiver/metadata.yaml

```
@@ -10,15 +10,87 @@ attributes:
   group:
     description: The ID (string) of a consumer group
     type: string
+  broker:
+    description: The ID (integer) of a broker
+    type: int
 
 metrics:
 #  brokers scraper
   kafka.brokers:
+    enabled: true
+    description: Number of brokers in the cluster (Deprecated).
+    unit: "{brokers}"
+    gauge:
+      value_type: int
```

Please add this line and regenerate. It'll automatically show the deprecation message if the metric is enabled
```suggestion
      value_type: int
    warnings:
      if_enabled: The metric is deprecated and will be removed. Use `kafka.brokers.count` instead
```